release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -2997,6 +2997,8 @@ link:https://access.redhat.com/solutions/6964873[{product-title} 4.10.20 contain
 
 * Previously, Bond Container Network Interface (CNI) version 1.0 was not compatible with the Multus Container Network Interface (CNI) plug-in. Consequently, the Bond-CNI IP address management (IPAM) improperly populated the `network-status` annotation. With this update, IPAM and Bond-CNI now supports Bond-CNI 1.0. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2084289[*BZ#2084289*])
 
+* Before this update, the Start Pipeline page was always created using a hard-coded value, `gp2`, as the default storage class. With this update, the Start Pipeline page uses the correct storage class name instead of the hard-coded value, `gp2`.
```

Comment:
```suggestion
* Before this update, the *Start Pipeline* page displayed `gp2` as the storage class, regardless of the actual storage class used. With this update, the *Start Pipeline* page displays the actual storage class name.
```
I believe this is your intended meaning.

Reply:
@rolfedh - Indeed, this is what I have meant from the start. Thank you for providing such valuable feedback. I have gone ahead and incorporated the changes. Kindly have a look and let me know if it looks good to you.